### DISCUSSION TITLE
能代替nas吗

### DISCUSSION THREAD

**Discussion by m986883511** [AlistGo/alist#4648] [Q&A]
有个疑问，为啥好多人在nas里面装alist，alist不是覆盖了nas上传下载最基本的功能了吗？如果我只需要上传下载最基本的功能，是不是不用搞nas那么麻烦。

职业：openstack开发
用过公司的黑群晖，如果alist只用本地存储的话，和nas是不是没啥区别？
我在家里用fedora虚拟机直通一块硬盘是不是就能当nas用了（我会搭建虚拟机局域网）

**Comment by BenzLeung** [comment]
我没有nas，我用了一台几百块钱的小盒子+两个usb大硬盘充当了nas，省了不少钱。所以我需要装个alist实现上传下载。

另外，alist有个重要功能，就是集成了市面上各种网盘和对象存储。

**Comment by sean19** [comment]
个人感觉不稳定

**Reply by m986883511** [reply]
确实不稳定
